react-frontend/src/components/Guides/CollaborationTools/collaborationTools.js

```diff
@@ -1,17 +1,55 @@
 import React from 'react';
-
+import styled from 'styled-components';
+import { createGlobalStyle } from 'styled-components';
 import CollaborationTool from './collaborationTool';
-// import ResourceHeader from './resourceheader';
-// import '@bcgov/bc-sans/css/BCSans.css'
-import '../../../css/collaborationTools.css';
+
+const GlobalStyle = createGlobalStyle`
+ html {
+  font-size: 10px;
+ }
+`;
+const CollabToolContainer = styled.div`
+  padding-top: 120px;
+  font-size: 2rem;
+  max-width: 80rem;
```

Comment:
same thing I think you want to avoid rem units for max width. It is less clear how large that it is. Is this opt to change a lot to be relative?

Reply:
This was one of the few things I put in that wasn't from word press.  The readability criteria on the max width was on number of words per line, not width.  Meaning font size seemed like the natural width unit.